pkg/providers/docker/docker.go

```diff
@@ -139,6 +139,11 @@ func (d *DockerTemplateBuilder) EtcdMachineTemplateName(clusterName string) stri
 	return fmt.Sprintf("%s-etcd-template-%d", clusterName, t)
 }
 
+func (vs *DockerTemplateBuilder) KubeadmConfigTemplateName(clusterName, workerNodeGroupName string) string {
+	t := vs.now().UnixNano() / int64(time.Millisecond)
+	return fmt.Sprintf("%s-%s-kubeadmconfig-template-%d", clusterName, workerNodeGroupName, t)
```

Comment:
the name feels too long, maybe just adding a timestamp?

Reply:
removing the `kubeadmconfig` and just adding `template`

Reply:
https://github.com/aws/eks-anywhere/pull/1175/commits/11c985eaa284af33896e87c73aa29e55be453b1e